st2reactor/st2reactor/sensor/base.py

```diff
@@ -16,6 +17,13 @@ class BaseSensor(object):
     """
 
     def __init__(self, sensor_service, config=None):
+        """
+        :param sensor_service: Sensor Service instance.
+        :type sensor_service: ``SensorService``
```

Comment:
Nice thing, added that as well: https://libcloud.readthedocs.org/en/latest/development.html#docstring-conventions

However curious, is there any difference?

Also is there any difference between single and double quotes: 
```
``dict``
```
and
```
`dict`
```
?